src/lib/mina_base/token_id_legacy.ml

```diff
@@ -0,0 +1,3 @@
+[%%import "/src/config.mlh"]
+
+include Mina_numbers.Token_id
```

Comment:
this file doesn't seem to be used anywhere - shouldn't we remove it?

Reply:
:+1: